Core/tooling/fix-bundles.js

```diff
@@ -85,9 +86,11 @@ openpgpLib = replace( // rsa verify on host
   const computed = await coreHost.verifyRsaModPow(m.toString(10), e.toString(10), n.toString(10)); // returns empty str if not supported: js fallback below
   const EM = computed ? new _bn2.default(computed, 10).toArrayLike(Uint8Array, 'be', n.byteLength()) : await _public_key2.default.rsa.verify(m, n, e);`
 );
+*/
 
 let openpgpLibBare = openpgpLib; // further modify bare code below
 
+/*
 openpgpLibBare = replace( // bare - produce s2k (decrypt key) on host (because JS sha256 implementation is too slow)
   openpgpLibBare,
```

Comment:
Same here ok for a quick fix to get build working, but will need to be fixed after.

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1379